data/json/monster_factions.json

```
@@ -406,6 +406,11 @@
     "neutral": [ "bot", "animal" ],
     "hate": [ "zombie", "fungus", "cult", "triffid", "nether" ]
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "robofac_spy",
+    "friendly": [ "human", "player" ]
```

```suggestion
    "base_faction": "robofac",
    "friendly": [ "human", "player" ]
```
I am uncertain if it would ever use its "special attack" in any event with FRIENDLY towards the player; perhaps "neutral" if tests indicate this is a problem?

---

They do dissapear properly, even without targets